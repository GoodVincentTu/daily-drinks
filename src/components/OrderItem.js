import React from 'react';

const OrderItem = (props) => {
  return (props.drink && 
    <>
      <li>
        <p>Name: {props.drink.title}</p>
        <p>Price: {props.drink.price}</p>
        <p>Notes: {props.drink.notes}</p>
      </li>
    </>
  )
};

export default OrderItem;