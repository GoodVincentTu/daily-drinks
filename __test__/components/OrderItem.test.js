import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import OrderItem from '../../src/components/OrderItem';

const fakeDrink = {
  title: '熟成紅茶',
  price: 20,
  notes: '微糖微冰'
};

describe('<OrderItem/>', () => {
  it('renders', () => {
    shallow(<OrderItem drink={fakeDrink}/>);
  });

  it('matches the snapshot', () => {
    const wrapper = shallow(<OrderItem/>);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});