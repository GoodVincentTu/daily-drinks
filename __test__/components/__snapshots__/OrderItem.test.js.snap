// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OrderItem/> matches the snapshot 1`] = `""`;
